client/src/ui/track/JobCard.tsx

```diff
@@ -245,13 +261,36 @@ export function JobCard({ application, profile }: JobCardProps) {
               padding: "6px",
               borderRadius: "50%",
             }}
-            onClick={(e) => e.stopPropagation()}
+            onClick={(e) => {
+              e.stopPropagation()
+              if (link) {
+                navigator.clipboard.writeText(link)
+                setAlert({
+                  message: "Link copied to clipboard",
+                  type: "success",
+                })
+              } else {
+                setAlert({
+                  message: "No link available",
+                  type: "error",
```

Comment:
Nit - Can we change the type to info or warning? This is not an error since the feature is not failing to execute

Reply:
gotcha